src/networktables/NetworkTable.cpp

```diff
@@ -43,6 +42,8 @@ void NetworkTable::SetTeam(int team) {
   char tmp[30];
 #ifdef _MSC_VER
   sprintf_s(tmp, "roboRIO-%d-FRC.local\n", team);
+#elif defined(__ANDROID__)
+  snprintf(tmp, 30, "roboRIO-%d-FRC.local\n",team);
```

Comment:
It would be nice if we could have another option in gradle that would let android be built without a gradle change. It would probably require a gradle option to specify the toolchain prefix, as different versions have different prefixes. However I don't know enough about gradle to set this up if we did end up wanting to do this.